server/service/src/sync/translations/asset_catalogue_type.rs

```diff
@@ -12,12 +12,12 @@ use super::{
 // Needs to be added to all_translators()
 #[deny(dead_code)]
 pub(crate) fn boxed() -> Box<dyn SyncTranslation> {
-    Box::new(AssetTypeTranslation)
+    Box::new(AssetCatalogueTypeTranslation)
 }
 
-pub(crate) struct AssetTypeTranslation;
+pub(crate) struct AssetCatalogueTypeTranslation;
 
-impl SyncTranslation for AssetTypeTranslation {
+impl SyncTranslation for AssetCatalogueTypeTranslation {
```

Comment:
it annoyed me looking for the file/struct that it didn't match the table name like every other translator